src/accounts/AccountPage.tsx

```diff
@@ -132,19 +142,34 @@ const AccountAboutPage: FC = () => {
             <CancellationOverlay onHideOverlay={hideDeactivateAccountOverlay} />
           </Overlay>
         </CancelServiceProvider>
+        {shouldShowDeleteFreeAccountButton && <DeleteFreeAccountButton />}
       </>
```

Comment:
Next week I think we need to do some digging to figure out what the difference is here - there's an extra button (deactivation) that doesn't show up for me in prod, and I think this is why:

- The code for account "deactivation" is hidden behind the `freeAccountCancellation` flag. It makes a `POST` request to /api/v2/quartz/cancel, which the openapi spec says is used to "cancel an account."

- The code for account "deletion" makes a `DELETE` request to /api/v2/quartz/account, which the API spec describes as being used to "self-delete an account." 

I'm not sure what the difference between these two is - deactivation and 'cancelation' get used interchangeably, and I'm not sure how that's different from deletion. So we should trace the quartz code on this to see whether those two things are different - maybe we only need one of these buttons if both of these endpoints are doing the same thing.